### ISSUE TITLE
train and test mAp

### ISSUE THREAD

**Issue by dsj1840923452** [Finspire13/CMCS-Temporal-Action-Localization#13]
hello, Thank you first for your code. I tested the thumos data with the model you gave me and found that the ap of the training set was much lower than that of the test set for the same kind of action.Below is the test result when iou=0.5. Red is the test set and green is the training set.What causes this?thank you

![zuozhe_model](https://user-images.githubusercontent.com/48936019/68097445-03c08480-fef2-11e9-8cca-3dfd7dd8af2f.png)

**Comment by Finspire13** [comment]
In weakly supervised setting, the results of training set are not necessarily better than the test set.
Because the training set only provide video-level classification label and not accurate boundaries.

I _guess_ the video-level classification accuracy will be higher on the training set.

**Comment by dsj1840923452** [comment]
Thank you but, theoretically, for the same category, the training should be similar to the testing. Now for the "golfswing" category, the test set has twice as many indicators as the training set.I looked at the standard deviation of the duration of this type of action and found that the training set was 17.92 and the test set was 7.76. Is this the reason?

**Comment by Finspire13** [comment]
Sorry, I am away from this project for a while and cant remember the dataset specifics. Yes, the gap between training and test set is an explanation. Maybe you can also visualize the results using [the code](https://github.com/Finspire13/CMCS-Temporal-Action-Localization/blob/master/utils.py#L805
) provided for a check.

**Comment by dsj1840923452** [comment]
ok，thank you
